common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -5398,6 +5398,12 @@
     },
     "sqlState" : "42902"
   },
+  "UNSUPPORTED_PARTITION_TRANSFORM" : {
+    "message" : [
+      "Unsupported partition transform: <transform>. Ensure that the transform expression is valid."
```

Comment:
The error message confuses slightly. Is the transform "unsupported" and "invalid". Can you help users and show which transform is expected: identity, bucket or clusterby.

Reply:
Improved the error message with specific transform names. Thansk!